btf/marshal.go

```diff
@@ -464,6 +483,41 @@ func (e *encoder) deflateEnumValues(values []EnumValue) ([]btfEnum, error) {
 	return bes, nil
 }
 
+func (e *encoder) deflateEnum64(raw *rawType, enum *Enum) (err error) {
+	if e.ReplaceEnum64 {
```

Comment:
Let's add a quick explanation with some context here. (e.g. why a union and not simply an int like we discussed offline)

Reply:
Fair point, but I'll add it to the commit message instead. I think the comment would be too large.